### ISSUE TITLE
Safe does not automatically unseal a vault it initializes

### ISSUE THREAD

**Issue by smgoller** [starkandwayne/safe#188]
safe 1.32, vault 1.2.2.

After deploying a safe manifest via bosh, running `safe init` does not automatically unseal the vault.

Output:

```
 $ safe vault status
Key                Value
---                -----
Seal Type          shamir
Initialized        false
Sealed             true
Total Shares       0
Threshold          0
Unseal Progress    0/0
Unseal Nonce       n/a
Version            n/a
HA Enabled         true
!! exit status 2
 $ safe init --json >safe.keys.json
 $ safe vault status
Key                Value
---                -----
Seal Type          shamir
Initialized        true
Sealed             true
Total Shares       5
Threshold          3
Unseal Progress    0/3
Unseal Nonce       n/a
Version            1.0.2
HA Enabled         true
!! exit status 2
```

**Comment by jhunt** [comment]
Can you retry the `safe init` call ith `DEBUG=1` set?

```
DEBUG=1 safe init --json >safe.keys.json
```

**Comment by smgoller** [comment]
The keys are unimportant, I spun up a local vault server for testing.

```
sean@develop-vm:~$ safe vault status                                            
Key                Value                                                        
---                -----                                                        
Seal Type          shamir                                                       
Initialized        false                                                        
Sealed             true                                                         
Total Shares       0                                                            
Threshold          0                                                            
Unseal Progress    0/0                                                          
Unseal Nonce       n/a                                                          
Version            n/a                                                          
HA Enabled         true                                                         
!! exit status 2                                                                
sean@develop-vm:~$ DEBUG=1 safe init --json >safe.keys.json                     
Request:                                                                        
PUT /v1/sys/init HTTP/1.1                                                       
Host: 127.0.0.1:8200                                                            
                                                                                
{"secret_shares":5,"secret_threshold":3,"root_token_pgp_key":"","pgp_keys":null}
                                                                                
Response:                                                                       
HTTP/1.1 200 OK
Content-Length: 649
Cache-Control: no-store
Content-Type: application/json
Date: Mon, 16 Sep 2019 17:21:09 GMT

{"keys":["96f32af8fb55a49213748c4797d7955aa569774e281f1e983992bb26eabf5d187c","5
13854ddf4f41d3a757fa47d5cd94c1958dbeb066247f8a474638d8a90bc00150d","762305cfed27
b73ed1756be3b841db78e3dff474da8dc497c6d983d1247516b463","18442fc367dbc961a125a1f
cd3b8c379a958849ad96d68b658ced0efa492df9713","f9b12c6f7fc9c7e690bdfa764c363fefaa
813576d93a1b980831ba2cc2c532136a"],"keys_base64":["lvMq+PtVpJITdIxHl9eVWqVpd04oH
x6YOZK7Juq/XRh8","UThU3fT0HTp1f6R9XNlMGVjb6wZiR/ikdGONipC8ABUN","diMFz+0ntz7RdWv
juEHbeOPf9HTajcSXxtmD0SR1FrRj","GEQvw2fbyWGhJaH807jDealYhJrZbWi2WM7Q76SS35cT","+
bEsb3/Jx+aQvfp2TDY/76qBNXbZOhuYCDG6LMLFMhNq"],"root_token":"s.bKyQWe3liUneS7ItEv
ACOnDq"}

sean@develop-vm:~$ safe vault status
Key                Value
---                -----
Seal Type          shamir
Initialized        true
Sealed             true
Total Shares       5
Threshold          3
Unseal Progress    0/3
Unseal Nonce       n/a
Version            1.2.2
HA Enabled         true
!! exit status 2
sean@develop-vm:~$
```

**Comment by thomasmitchell** [comment]
heh. The JSON flag is causing it to bail before it gets to the unseal code.

**Comment by thomasmitchell** [comment]
8c56a180835a1af561c8ec32203ba9346fb6a7ef

**Comment by thomasmitchell** [comment]
Fixed in v1.3.4
